felix/fv/infrastructure/datastore_describe.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
```

Comment:
nit: it seems recently, we prefer to use `logrus` instead of using `log` alias for sake of clarity.